.github/workflows/build_and_test.yml

```diff
@@ -428,3 +428,41 @@ jobs:
     - name: Build with SBT
       run: |
         ./build/sbt -Pyarn -Pmesos -Pkubernetes -Phive -Phive-thriftserver -Phadoop-cloud -Pkinesis-asl -Phadoop-2.7 compile test:compile
+
+  tpcds-1g:
+    name: Run TPC-DS queries with SF=1
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout Spark repository
+      uses: actions/checkout@v2
+    - name: Checkout TPC-DS (SF=1) generated data repository
+      uses: actions/checkout@v2
+      with:
+        repository: maropu/spark-tpcds-sf-1
+        path: ./tpcds-sf-1
```

Comment:
@dongjoon-hyun Is it acceptable to use a personal repository?

Reply:
Either https://github.com/maropu/spark-tpcds-datagen or https://github.com/maropu/spark-tpcds-datagen doesn't look having a license problem. I don't see any problem of using it because we're already using other repos in our CI. GA plugins, or other personal repos such as https://github.com/apache/spark/blob/master/dev/appveyor-install-dependencies.ps1#L98

Reply:
Thank you for pinging me, @wangyum . Ya, I hope we can bring this into Apache Spark eventually.  For now, let's move on. I also need to move GitHub Action linux image into Apache Spark repo in this year.

Reply:
Thank you @dongjoon-hyun

Reply:
Yea, I also think it is the best to move the repo into the official one finally.